Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -122,6 +122,43 @@ public override void Initialize()
                 CameraCache.Main.transform.position = Vector3.zero;
                 CameraCache.Main.transform.rotation = Quaternion.identity;
 
+                // Ensure there's an event system attached to the main camera.
+                bool addedComponents = false;
```

Comment:
This feels like a major behavior change (it doesnt look like this code was moved from elsewhere in the input system).

It also does not feel like the responsibility of the input system to enforce this (read: what if the input system is not being used?).

Reply:
Shouldnt this be the responsiblity of the camera system?

Reply:
The camera system doesn't know or care about input, so it felt odd for the camera system to handle the business of managing input-related components. Imagine a dev writing a custom implementation of the camera service - it wouldn't naturally occur to them to include this step when creating their main camera.

>It also does not feel like the responsibility of the input system to enforce this (read: what if the input system is not being used?).

This is true, if the input system isn't being used then the event system is not added. That seemed like a good thing to me, since if I've shut off the input system I assume I'd be doing it to have total control over what input-related components are being added to the scene. Are there are side effects I'm not considering here? Ie does anything wonky happen if the input system is disabled and there's no event system added to the scene?

Reply:
> if the input system isn't being used then the event system is not added

In the current implementation, which component is adding the event system to the camera?

Reply:
The MRTK instance adds it under `EnsureMixedRealityRequirements`.